Index.html

```diff
@@ -31,7 +31,170 @@ <h3 class="connect">LET’S CONNECT</h3>
         </ul>
       </div>
   </section>
-    
+   <section class="experience">
+    <div>
+      <div class="proj1">
+        <div class="proj-content-space">
+          <h2 class="proj-title">Tonic</h2>
+          <nav>
+            <h4 class="cpny">Canopy</h4>
+            <img src="./images/Counter.png" alt="counter" />
+            <span class="project-lang">Back End Dev</span>
+            <img src="./images/Counter.png" alt="counter" />
+            <span>2015</span>
+          </nav>
+          <p class="proj-desc">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="softwares">
+            <li class="software">html</li>
+            <li class="software">css</li>
+            <li class="software">javascript</li>
+          </ul>
+          <button class="btn">see project</button>
```

Comment:
Kindly update your button element to include button interaction styling when `hovered ` and `pressed`🛠️. You can use `.button:hover` and `.button:active` to style such interactions. Please ensure to match the design guide provided by figma 🎨

Figma design:
<img width="359" alt="image" src="https://user-images.githubusercontent.com/37116322/219805104-27806d66-228a-4933-b1bb-1e3eb9011ec8.png">